epochtalk/epochtalk
# Recent Posted In

Created a new routes, views, and permissions for viewing recent threads
the viewer has posted in. This takes advantage of a new db method
created in the sister branch in core-pg. This page is accessible through
a link on the front page, in the recent threads section or through the
url '/threads/posted'

This commit also refactors some of the angular code so that each module
or view is in their own folders rather than being lumped into the same
folder.

Testing Procedures:
- Log in
- Go the '/'
- Click on the 'Threads Posted In' Link
- Ensure your posts are there in order and that you can go to them
- Log Out
- Ensure you can't view the same page ('/threads/posted')
- Log in again
- Make a new post
- Ensure post is at the top of the 'Threads Posted In' page.
- Delete or Purge the post
- Ensure the thread no longer shows up
